translations/ui-courses/en.json

```diff
@@ -192,5 +192,10 @@
     "addFastAddItem.title": "Add Fast Add item",
     "addFastAddItem.tooltip": "Create a new Inventory record using 'Fast Add' and add it to this course as a reserve",
     "errors.mustBeInteger": "This value must be an integer",
-    "SENTINEL": "SENTINEL"
+    "SENTINEL": "SENTINEL",
```

Comment:
Please make sure SENTINEL remains the last entry. (That's what it's for.)

Reply:
Done!